lib/ood_core/app_config.rb

```diff
@@ -0,0 +1,168 @@
+require "ood_core/refinements/array_extensions"
+require 'pathname'
+require 'yaml'
+
+module OodCore
+    class AppConfig
+        using Refinements::ArrayExtensions
+
+        attr_reader :raw_config
```

Comment:
Why call it `raw_config` instead of just `config`?

Reply:
`raw_config` signifies that it has not been processed. No defaults are set and no string-paths have been converted to `Pathnames`.

Reply:
Part of the idea was to be able to get at the data before it was transformed in case the transformation results in lost information.

Reply:
Sounds good